lib/tasks/sidekiq/cleanup.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'three_scale/sidekiq_batch_cleanup_service'
+
+namespace :sidekiq do
+  desc 'cleanup BID-* keys from sidekiq-batch, specify the max age in seconds as an argument'
+  task :cleanup_batches, [:max_age_seconds] => :environment do |task, args|
+    max_age_seconds = args[:max_age_seconds].to_i
```

Comment:
```suggestion
    max_age_seconds = Integer(args[:max_age_seconds])
```

`#to_i` will not fail even if some garbage has been supplied while `Integer()` will raise on any unexpected chars. I think in this case it is safer to only run with fully correct arguments.